app/src/boardgames/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from django.utils.safestring import mark_safe
+
+
+class BoardgameForm(forms.Form):
+    url = forms.CharField(
+        label=mark_safe('Find your boardgame on: <a href="https://boardgamegeek.com/">\
+            boardgamegeek.com</a> and paste here link'), max_length=200)
```

Comment:
I'm not sure whether Python will put indentation spaces into the string, so I think the feature of implicit string concatenation should be used here.